pkg/input/container/docker.go

```diff
@@ -170,50 +173,53 @@ func (dt *DockerTailer) forwardMessages() {
 			return
 		}
 
-		ts, updatedMsg := dt.updatedDockerMessage(msg.Content())
+		ts, sev, updatedMsg := dt.parseMessage(msg.Content())
+		dt.checkForNewDockerTags()
```

Comment:
doing this for every message, we take into account the fact that docker tags _can_ change. This should be cached so not very expensive, despite the fact that we need to compare arrays